languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8521,32 +8521,61 @@ though, and has since been slightly modified)-->
             <example><marker>C’est</marker> une belle maison.</example>
             <example correction="C'est"><marker>Cet</marker> un garçon !</example>
         </rule>
-        <rule id="CEST_CETTE" name="C’EST au lieu de CETTE CET ou CES">
-            <pattern>
-                <token>
-                    <exception>,</exception></token>
-                <marker>
-                    <token>c'</token>
-                    <token>est</token>
-                </marker>
-                <token postag="N.*" postag_regexp="yes">
-                    <exception regexp="yes">pas|dommage|grâce|hyper|monsieur|madame|mademoiselle|oui|non|lundi|mardi|mercredi|jeudi|vendredi|samedi|dimanche|bidon|bonnet|la|suite|incognito|face|ballot</exception>
-                    <exception postag="[^N].*" postag_regexp="yes"/></token>
-            </pattern>
-            <message>Voulez-vous écrire <suggestion>ces</suggestion>, <suggestion>cet</suggestion> ou <suggestion>cette</suggestion> au lieu de « \2\3 » ?</message>
-            <example correction="Ces|Cet|Cette"><marker>C’est</marker> école est grande.</example>
-            <example><marker>Cette</marker> école est réputée.</example>
-            <example>C’est dommage de partir si tôt.</example>
-            <example>Vouloir c’est pouvoir.</example>
-            <example>C’est non seulement pour lui, mais aussi pour elle.</example>
-            <example>C’est dimanche qu’il arrive.</example>
-            <example>Comme c’est romantique.</example>
-            <example>C’est la fin.</example>
-            <example>C’est suite à cette victoire qu’il fut élu.</example>
-            <example>C’est incognito qu’il reporta le prix.</example>
-            <example>C’est face à la montée en puissance.</example>
-            <example>C'est ballot !</example>
-        </rule>
+        <rulegroup id="CEST_CETTE" name="C’EST au lieu de CETTE CET ou CES">
```

Comment:
The old rule is so bad, this one can replace it and still go "temp_off" at first